azure-pipelines.yml

```diff
@@ -0,0 +1,84 @@
+trigger:
+- dev
```

Comment:
what about master branch?

Reply:
Didn't add that yet since I wanted to test it out on the dev branch before pushing it out to the master. Now that you mention it: Would be better to add it in this PR before merging into master so that changes goes into master. I'll add it in.